packages/react/src/lib/useVoiceClient.ts

```diff
@@ -134,23 +137,26 @@ export const useVoiceClient = (props: {
                 content: string;
               }) => ({
                 type: 'tool_error',
-                tool_call_id: message.tool_call_id,
+                toolCallId: message.toolCallId,
                 error,
                 code,
-                level,
+                level: level != null ? 'warn' : undefined, // level can only be warn
```

Comment:
change `!=` to `!==`?

Reply:
i think `!=` is what we want (e..g if level is `undefined` we probably don't want to default to `warn`)

Reply:
If we know in advance that the value could be `undefined`, we should check against `!level` instead. Intended behavior can be misconstrued when we purposely rely on Javascript coercion

Reply:
fwiw, i agree with Yini here. Relying on strict equality is less error prone.